.github/CODEOWNERS

```diff
@@ -26,6 +26,10 @@
 ####
 # Service team
 ####
+
+# PRLabel: %AgriFood
+/sdk/agrifood/azure-agrifood-farming                                 @bhargav-kansagara
```

Comment:
@kristapratico:  Can you help us understand which of these are service team contacts that need to be mentioned when the `Service Attention` label is applied?

It looks like that would be:

- ArgiFood
- AppInsights
- Monitor Log Analytics
- Communication (all)
- Attestation
- Confidential Ledger
- Load Testing

Is that right?

Reply:
Okay updated with the Service Attention label. I put the actual path to the library instead of NotInRepo since these libraries actually are in our repo.... but I didn't see this anywhere else in the file. Is that right?